price-feeder/config/config.go

```diff
@@ -52,6 +52,16 @@ var (
 	// maxDeviationThreshold is the maxmimum allowed amount of standard
 	// deviations which validators are able to set for a given asset.
 	maxDeviationThreshold = sdk.MustNewDecFromStr("3.0")
+
+	// SupportedQuotes defines a lookup table for which assets we support
+	// using as quotes.
+	SupportedQuotes = map[string]struct{}{
+		DenomUSD: {},
+		"USDC":   {},
+		"USDT":   {},
+		"BTC":    {},
+		"ETH":    {},
```

Comment:
should Atom be here as well?

Reply:
DAI also

Reply:
DAI yes, atom we're not adding until we can measure volume ( ref #1117 )

Reply:
c332a4b71c9d7318e052723cc637cfc1a0d98641